models/stablelm/stablelm-jp-3b-ja50_rp50-700b/result.json

```diff
@@ -1,48 +0,0 @@
-{
```

Comment:
Note: this file was renamed to `models/stablelm/stablelm-jp-3b-ja50_rp50-700b/result_template-0.2.json`